### PR TITLE
2023 autumn training

### PR DESCRIPTION
**Pull request by patricia-ternes**
This PR creates a blog with all dates and info related with our Autumn Courses.

## Minor fix

It also includes a variable fix in the SWD1b page. In the moment is not loading any inoformation from the `_data/training_courses.yml` file.

### CODE REVIEW

**Review by murphyqm — COMMENTED** [review]
Have read through and checked all the links, everything is working; didn't spot any typos. Just to note the [swd 1b page](https://arc.leeds.ac.uk/training/courses/swd1b/) is currently empty

Edit - spotted the variable fix to load content onto the SWD1b page.

File: blog/_posts/2023-09-22-Autumn2023TrainingCourses.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+layout: page
+title:  Spring 2023 Training Courses now open
```
Comment (jhodrien) [inline blog/_posts/2023-09-22-Autumn2023TrainingCourses.md:3]:
```suggestion
title:  Autumn 2023 Training Courses now open
```

File: blog/_posts/2023-09-22-Autumn2023TrainingCourses.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+date:   2023-01-30
```
Comment (jhodrien) [inline blog/_posts/2023-09-22-Autumn2023TrainingCourses.md:4]:
```suggestion
date:   2023-09-22
```

File: blog/_posts/2023-09-22-Autumn2023TrainingCourses.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+breadcrumb: true
+---
+
+We've now confirmed some upcoming training courses which are now open for registrations!
+
+Whether you want to learn some Git, Linux command line or learn more about the University's high performance computing systems we've got a course for you.
+They are hands-on so be prepared to do some typing!
+
+All booking is managed through SAP and by the IT Training team, any questions about your booking should be directed to ittraining@leeds.ac.uk.
```
Comment (jhodrien) [inline blog/_posts/2023-09-22-Autumn2023TrainingCourses.md:21]:
```suggestion
All booking is managed through SAP and by the IT Training team, any questions about your booking should be directed to [ittraining@leeds.ac.uk](mailto:ittraining@leeds.ac.uk).
```

**Review by murphyqm — APPROVED** [review]
No obvious typos, MD preview looks good, links are correct

File: pages/training/training_courses/swd1b.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ breadcrumb: true
 
 {% assign swd1bw = site.data.training_courses.widget | where:'code', 'SWD1b' %}
 {% assign swd1b = swd1b[0] %}
```
Comment (jhodrien) [inline pages/training/training_courses/swd1b.md:8]:
```suggestion
{% assign swd1b = swd1bw[0] %}
```

**Review by jhodrien — CHANGES_REQUESTED** [review]
Not checked the dates, as I think others have done this.

**Pull request merged**